### PR TITLE
fix: property pills are horizontally aligned now

### LINKED ISSUE #11131: Property Pills stack vertically in ActivityLog Sentence Lists
### viewed in feature flag

Screenshot 2022-08-04 at 09 56 42

### viewed in history tab

Screenshot 2022-08-04 at 09 53 56

When a change in the activity log has more than one property pill they stack vertically. They should stack horizontally. Ideally separated by commas using the `SentenceList` component.

## How to reproduce

* change a filter with more than one property in a feature flag
* view the activity log

## Environment

- [x] PostHog Cloud
- [x] self-hosted PostHog, version/commit: please provide

### PR DESCRIPTION
**Pull request by kevoconnell**
## Problem
- fixes #11131 

## Changes
Screen Shot 2022-08-11 at 2 25 13 AM
Screen Shot 2022-08-11 at 2 26 24 AM
Screen Shot 2022-08-11 at 2 27 57 AM

- doesn't affect property pill when onClose 
Screen Shot 2022-08-11 at 2 25 39 AM

## How did you test this code?

- create a feature flag.
- change the flag and have more than two conditions
- change to have less than two conditions
- change again to have no conditions
- enable or disable a flag
The alignment should look correct for all of these cases.

### CODE REVIEW

**Comment by benjackwhite** [comment]
Nice work! I think we need to look closely at the other use cases of the PropertyFilter component. It's used in a lot of crucial places and it's not super clear to me if this breaks things...

**Comment by kevoconnell** [comment]
> Nice work! I think we need to look closely at the other use cases of the PropertyFilter component. It's used in a lot of crucial places and it's not super clear to me if this breaks things...

thanks! from what I see PropertyFilter is used in experiments and insights, the latter looks good to me, but I can't access the former 😢 
Screen Shot 2022-08-11 at 1 24 56 PM

File: frontend/src/lib/components/ActivityLog/activityLogLogic.feature-flag.test.tsx [hunk 1]
```diff
@@ -364,7 +364,7 @@ describe('the activity log logic', () => {
             const actual = logic.values.humanizedActivity
 
             expect(render(<>{actual[0].description}</>).container).toHaveTextContent(
-                'peter changed the filter conditions to apply to 100% ofID 98, and 100% ofID 411 on with cohort'
+                'peter changed the filter conditions to apply to 100% of ID 98 , and 100% of ID 411 on with cohort'
```
Comment (pauldambra) [inline frontend/src/lib/components/ActivityLog/activityLogLogic.feature-flag.test.tsx:367]: good catch on these spaces 🙌

File: frontend/src/lib/components/PropertyFilters/components/PropertyFiltersDisplay.tsx [hunk 2]
```diff
@@ -11,8 +11,14 @@ const PropertyFiltersDisplay: React.FunctionComponent<Props> = ({ filters, style
...
         <div className="PropertyFilters mb-4" style={style}>
             {filters &&
-                filters.map((item) => {
-                    return <PropertyFilterButton key={item.key} item={item} />
+                filters.map((item, idx) => {
+                    return (
+                        <>
+                            {' '}
+                            <PropertyFilterButton style={{ margin: '0.1rem' }} key={item.key} item={item} />{' '}
+                            {idx !== filters.length - 1 ? ',' : ''}
```
Comment (pauldambra) [inline frontend/src/lib/components/PropertyFilters/components/PropertyFiltersDisplay.tsx:19]: From experience with our `PropertyFilter` I think this is the riskiest part of the change

I'd rather wrap an array of `PropertyFilter` in a `SentenceList` in the specific case(s) where they need to have commas so we remove the risk of commas in unexpected places

This is way more a criticism of our (lack of) visual testing of such a critical component than your PR!

**Review by pauldambra — CHANGES_REQUESTED** [review]
This is a solid change... just the one bit we could do with tweaking 🙌

File: frontend/src/lib/components/PropertyFilters/components/PropertyFiltersDisplay.tsx (hunk 2 above)
Comment (MrKevinOConnell) [reply frontend/src/lib/components/PropertyFilters/components/PropertyFiltersDisplay.tsx]: > This is a solid change... just the one bit we could do with tweaking 🙌

Here's what it looks like now! @pauldambra 
Screen Shot 2022-08-12 at 11 37 39 AM

**Comment by pauldambra** [comment]
* stripped out some changes to make the diff as small as possible
* set the sentences to wrap when wider than viewport
    * this does mean pills touch when wrapped but I think that's better than some things having a little margin bump

Screenshot 2022-08-25 at 22 55 06

**Review by pauldambra — APPROVED** [review]
I agree with myself 

spiderhogs-54f1bf6e8ebf31791e4c5c1813b19f2a

**Pull request merged**